syntax = "proto3";

package example;

service PromptService {
  rpc GetResponse (PromptRequest) returns (PromptReply) {}
  rpc RunAgent(AgentRequest) returns (AgentReply) {}
}

message PromptRequest {
  string prompt = 1;
}

message PromptReply {
  string answer = 1;
}

// Definitionen f√ºr RunAgent
message AgentRequest {
  string prompt = 1;
}

message AgentReply {
  string answer = 1;
}